graveyard.json

```
@@ -2262,5 +2262,13 @@
     "link": "https://support.grasshopper.app/t/grasshopper-is-shutting-down-on-june-15-2023/61994",
     "description": "Grasshopper was a free mobile and web app for aspiring programmers that taught introductory JavaScript and coding fundamentals using fun, bite-sized puzzles.",
     "type": "app"
+  },
+  {
+    "name": "Glass Enterprise Edition",
+    "dateOpen": "2017-07-01",
+    "dateClose": "2023-09-15",
+    "link": "https://support.google.com/glass-enterprise/customer/answer/13417888",
```

> **Please do not use Google Support articles, linking to the product's URL, the product's marketing URL, or other Google-provided links.*

I suggest that we should use [this link](https://www.pcmag.com/news/rip-google-glass-enterprise-as-sales-for-smart-glasses-end) instead as Google Support links are not allowed here.

```suggestion
    "link": "https://www.pcmag.com/news/rip-google-glass-enterprise-as-sales-for-smart-glasses-end",
```